admin/kernel-masterdata-service/src/test/java/io/mosip/kernel/masterdata/test/utils/MasterdataSearchHelperTest.java

```diff
@@ -246,28 +263,481 @@ public void searchwildcardSearchMasterdata() {
+	public void searchMasterdataWithoutLangCode07() {
+		SearchDtoWithoutLangCode searchDtoWithoutLangCode = new SearchDtoWithoutLangCode(Arrays.asList(noBetweenValueFilter), Arrays.asList(sort), page, "eng");
+		searchHelper.searchMasterdataWithoutLangCode(RegistrationCenter.class, searchDtoWithoutLangCode, optionalFilterArray);
+	}
+	
+	@Test(expected = RequestException.class)
+	public void searchMasterdataWithoutLangCode08() {
+		SearchDtoWithoutLangCode searchDtoWithoutLangCode = new SearchDtoWithoutLangCode(Arrays.asList(noColumnFilter), Arrays.asList(sort), page, "eng");
+		searchHelper.searchMasterdataWithoutLangCode(RegistrationCenter.class, searchDtoWithoutLangCode, optionalFilterArray);
+	}
+	
+	@Test(expected = RequestException.class)
+	public void searchMasterdataWithoutLangCode09() {
+		SearchDtoWithoutLangCode searchDtoWithoutLangCode = new SearchDtoWithoutLangCode(Arrays.asList(noValuefilter), Arrays.asList(sort), page, "eng");
+		searchHelper.searchMasterdataWithoutLangCode(RegistrationCenter.class, searchDtoWithoutLangCode, optionalFilterArray);
+	}
+	
+	@Test(expected = RequestException.class)
+	public void searchMasterdataWithoutLangCode10() {
+		SearchDtoWithoutLangCode searchDtoWithoutLangCode = new SearchDtoWithoutLangCode(Arrays.asList(filter), Arrays.asList(noColumnSort), page, "eng");
+		searchHelper.searchMasterdataWithoutLangCode(RegistrationCenter.class, searchDtoWithoutLangCode, optionalFilterArray);
+	}
+	
+	@Test(expected = RequestException.class)
+	public void searchMasterdataWithoutLangCode11() {
+		SearchDtoWithoutLangCode searchDtoWithoutLangCode = new SearchDtoWithoutLangCode(Arrays.asList(filter), Arrays.asList(noColumnSort), page, "eng");
+		searchHelper.searchMasterdataWithoutLangCode(RegistrationCenter.class, searchDtoWithoutLangCode, optionalFilterArray);
+	}
+	
+	@Test(expected = RequestException.class)
+	public void searchMasterdataWithoutLangCode12() {
+		SearchDtoWithoutLangCode searchDtoWithoutLangCode = new SearchDtoWithoutLangCode(Arrays.asList(noFilterType), Arrays.asList(noColumnSort), page, "eng");
+		searchHelper.searchMasterdataWithoutLangCode(RegistrationCenter.class, searchDtoWithoutLangCode, optionalFilterArray);
+	}
+	
+	@Test
+	public void searchMasterdataWithoutLangCode13() {
+		sort.setSortType("desc");
+		SearchDtoWithoutLangCode searchDtoWithoutLangCode = new SearchDtoWithoutLangCode(Arrays.asList(filter), Arrays.asList(sort), page, "eng");
+		searchHelper.searchMasterdataWithoutLangCode(RegistrationCenter.class, searchDtoWithoutLangCode, optionalFilterArray);
+	}
+	
+	@Test
+	public void searchMasterdataWithoutLangCode14() {
+		sort.setSortType("asc");
+		SearchDtoWithoutLangCode searchDtoWithoutLangCode = new SearchDtoWithoutLangCode(Arrays.asList(filter), Arrays.asList(sort), page, "eng");
+		searchHelper.searchMasterdataWithoutLangCode(RegistrationCenter.class, searchDtoWithoutLangCode, optionalFilterArray);
+	}
+	
+	@Test
+	public void getMissingDataTest(){
```

Comment:
for positive test cases, use assertions wherever possible

Reply:
Most of the methods are private only, added for possible public methods